resources/axon.rb

```diff
@@ -58,7 +58,10 @@
 
   # Set platform specific variables
   case node['platform']
-  when 'centos', 'redhat', 'suse', 'oraclelinux', 'oracle', 'amazon'
+  when 'centos', 'redhat', 'suse', 'oraclelinux', 'amazon'
+    ext = '.rpm'
+    eg_service_name = 'tw-eg-service'
```

Comment:
This code ends up being duplicated. Can we do something similar to the last one where we make the second piece if platform is oracle instead of duplicating these two lines?

Reply:
Yes Kraig, Last time we have collaborated the oracle part in the upper when block and due to that change it started failing for 'Redhat 6.x' and 'Centos 6.x' bcoz "node["hostnamectl"]["kernel"]' property is not applicable on the these machines.
So according to us oracle specific code should be in different block so that it won't effect to the other platforms.